src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -404,7 +403,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         final var startClock = System.nanoTime();
 
-        if (StringUtils.isEmpty(this.encoding)) {
+        if (this.encoding == null || this.encoding.length() == 0) {
```

Comment:
:+1: to this change, if it avoids a whole dependency

Reply:
will pull this one out and separately apply by itself.